examples/example-gcp-observability/README.md

```diff
@@ -0,0 +1,39 @@
+gRPC GCP Observability Example
+================
+
+The GCP Observability example consists of a Hello World client and a Hello World server instrumented for logs, metrics and tracing. 
+
+__Please refer to GCP observability user guide for setting up authorization to acess Google Cloud Platform with Google user credentials.__
```

Comment:
> ...acess Google Cloud Platform with Google user credentials.__

1) s/acess/access/

2) Are there user credentials or service account credentials 

3) For the simplest case of using the current project as destination project is there anything special needed?

Reply:
To avoid repetition of instructions in both user guide and here, updated to refer to user guide instead. Let me know if you think we need more detail here.

If the `project-id` can be derived from the environment, it works (yet to be verified from a GCE VM).